src/Microsoft.Fast.Components.FluentUI/DesignTokens/DesignToken.razor.cs

```diff
@@ -0,0 +1,143 @@
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.Extensions.Configuration;
+using Microsoft.JSInterop;
+
+namespace Microsoft.Fast.Components.FluentUI.DesignTokens;
+
+public partial class DesignToken<T> : ComponentBase, IDesignToken<T>, IAsyncDisposable
+{
+    [Inject]
+    private IJSRuntime JSRuntime { get; set; } = default!;
+
+    [Inject]
+    private IConfiguration Configuration { get; set; } = default!;
+
+    private Lazy<Task<IJSObjectReference>>? moduleTask;
+    private Reference Target { get; set; } = new Reference();
+
+    private T? _defaultValue;
+
+    /// <summary>
+    /// Gets the name of this design token 
+    /// </summary>
+    public string? Name { get; init; }
+
+    /// <summary>
+    /// Gets or sets the value of the design token
+    /// </summary>
+    [Parameter]
+    public T? Value { get; set; }
+
+    /// <summary>
+    /// Gets or sets the content to apply this design token on
+    /// </summary>
+    [Parameter]
+    public RenderFragment<Reference>? ChildContent { get; set; }
+
+    /// <summary>
+    /// Constructs an instance of a DesignToken.
+    /// </summary>
+    public DesignToken()
+    {
+
+    }
+
+    /// <summary>
+    /// Constructs an instance of a DesignToken.
+    /// </summary>
+    public DesignToken(IJSRuntime jsRuntime, IConfiguration configuration)
+    {
+        JSRuntime = jsRuntime;
+        Configuration = configuration;
+
+        Initialize();
+    }
+
+    /// <inheritdoc/>
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        if (firstRender && Value is not null)
+        {
+            Initialize();
+
+            await SetValueFor(Target.Current, Value);
+            StateHasChanged();
+        }
+    }
+
+    private void Initialize()
+    {
+        string scriptSource = Configuration["FluentWebComponentsScriptSource"] ?? "https://cdn.jsdelivr.net/npm/@fluentui/web-components/dist/web-components.min.js";
```

Comment:
Is there a convenient way to make this default a config driven value rather than hardcoded? Or should it just error if the value is not in the Configuration?

Reply:
It is a config driven value now. You can set it in the appsettings.json file (with the key as shown above). If a value is not supplied, we'll use the default location as described in the readme for the setup on the library (the jsdeliver link)